.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug or performance issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+The following information might be useful: 
+ - CMake command: [e.g.cmake -DCMAKE_CXX_COMPILER=icpx -DCMAKE_BUILD_TYPE=release -DONEDPL_PAR_BACKEND=tbb]
+ - oneDPL version: [e.g. 2022.6.0 or 224acf7423ded15d34b53429d99248c3d308b685] <br/>
+    Report oneDPL version and githash. If it is a regression, report githash for the last known good revision.
```

Comment:
Please cite githash as GitHash.

Reply:
The word isn't referring to a project, but is instead referring to the hash of a commit to the oneDPL repository.  I've replaced "githash" with "git hash".